src/test/java/com/znsio/sample/e2e/screen/android/amazonsearch/AmazonHomeScreenAndroid.java

```diff
@@ -0,0 +1,52 @@
+package com.znsio.sample.e2e.screen.android.amazonsearch;
+
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonHomeScreen;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonSearchResultsScreen;
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.MobileElement;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class AmazonHomeScreenAndroid extends AmazonHomeScreen {
+    private final Driver driver;
+    private final Visual visually;
+    private static final String SCREEN_NAME = AmazonHomeScreenAndroid.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    public AmazonHomeScreenAndroid(Driver driver, Visual visually) {
+        super();
+        this.driver = driver;
+        this.visually = visually;
+        visually.checkWindow(SCREEN_NAME, "Home Page");
+    }
+
+    public static final By bySearchBoxXpath = By.xpath("//android.view.View[@content-desc='Clear search keywords']");
```

Comment:
is there any reason by is added before each variable name

Reply:
In Anand's demo of android automation, variables were named like this.